nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/connection-details.jsp

```diff
@@ -39,7 +39,6 @@
                     <div id="read-only-relationship-names-container" class="setting">
                         <div class="setting-name">
                             Relationships
-                            <div class="fa fa-question-circle" alt="Info" title="Selected relationships are in bold."></div>
```

Comment:
Should this label also be updated to `Selected Relationships` so there's no confusion over what we're showing?